src/GitHubExtension/DataManager/GitHubDataManager.cs

```diff
@@ -16,6 +16,7 @@ public partial class GitHubDataManager : IGitHubDataManager, IDisposable
     private static readonly TimeSpan NotificationRetentionTime = TimeSpan.FromDays(7);
     private static readonly TimeSpan SearchRetentionTime = TimeSpan.FromDays(7);
     private static readonly TimeSpan PullRequestStaleTime = TimeSpan.FromDays(1);
+    private static readonly TimeSpan ReviewStaleTime = TimeSpan.FromDays(7);

```

Comment:
At some point we should move all of these "configuration" type items to a central file maybe

Reply:
Agree and disagree. I agree with the sentiment of putting such constants and configuration data together, but disagree on basis of opportunity cost. Most of these are unlikely to be something we'd ever be changing all that often, so I'm not sure the development time/effort is worth the benefit unless it is created as part of something else, or we're exposing these to be highly configurable (which we might do in the future).  So I agree this would be good, but don't agree that it's worth doing unless we have a more tangible value prop for doing it or it becomes trivial to do.